src/dlgAboutDialog.cpp

```diff
@@ -94,4 +98,696 @@ dlgAboutDialog::dlgAboutDialog(QWidget* parent) : QDialog(parent)
+                               "<tt>Gnomovision version 69, Copyright (C) year name of author\n"
+                               "Gnomovision comes with ABSOLUTELY NO WARRANTY; for details\n"
+                               "type 'show w'.  This is free software, and you are welcome\n"
+                               "to redistribute it under certain conditions; type 'show c'\n"
+                               "for details.</tt>"
+                               "<p>The hypothetical commands <tt>'show w'</tt> and <tt>'show c'</tt> should "
+                               "show the appropriate parts of the General Public License.  Of course, the "
+                               "commands you use may be called something other than <tt>'show w'</tt> and "
+                               "<tt>'show c'</tt>; they could even be mouse-clicks or menu items--whatever "
+                               "suits your program.</p>"
+                               "<p>You should also get your employer (if you work as a programmer) or your "
+                               "school, if any, to sign a &quot;copyright disclaimer&quot; for the program, if "
+                               "necessary. Here is a sample; alter the names:</p>"
+                               "<tt>Yoyodyne, Inc., hereby disclaims all copyright\n"
+                               "interest in the program 'Gnomovision'\n"
+                               "(which makes passes at compilers) written\n"
+                               "by James Hacker.\n"
+                               "\n"
+                               "signature of Ty Coon, 1 April 1989\n"
+                               "Ty Coon, President of Vice</tt>"
+                               "<p>This General Public License does not permit incorporating your program into "
+                               "proprietary programs. If your program is a subroutine library, you may "
+                               "consider it more useful to permit linking proprietary applications with the "
+                               "library. If this is what you want to do, use the "
+                               "<a href=\"https://www.gnu.org/licenses/lgpl.html\">GNU Lesser General Public "
+                               "License</a> instead of this License.</p>"));
+
+    //    textBrowser_license->setFont(QFont(QStringLiteral("Bitstream Vera Serif"), 14));
+    textBrowser_license->setHtml(
+                QStringLiteral("<html>%1<body>%2<hr>%3</body></html>")
+                .arg(htmlHead, headerText, gplText));
+
+    // TAB 3 - Third party items
+    // Only the introductory text at the top and interspersed between is to be
+    // translated - the Licences themselves MUST NOT be translated:
+    QString thirdPartiesHeader(
+                tr("<h3>Mudlet makes use of some third-party <i>non</i>-GPL software libraries which are listed here with their license information:</h3>"));
```

Comment:
Would just going with `third-party software libraries` be an issue for you?

Reply:
Sorry, but yes it would, because the GPL ones specifically do NOT have to be listed as their licences do not require it and the only reason we are including these here is because we have to... (perhaps that sounds a bit harsh, but it took me a while to suss out the details, and these things being non-GPL makes it harder to use them and maintain a complete and full list of their terms of use) TBH I am not 100% certain I have nailed down all the licences but with the framework here it should be a little easier to insert further entries if needed.

Reply:
Yes, you've done a ton of work on this! Much appreciated :)

How about this positive-sounding stuff, because all of this is open-source at the end of the day and that is what really matters:

> Mudlet is built on the shoulders of giants. Here are some of the third-party non-GPL libraries we make use of:

Reply:
I can use:

> Mudlet is built on the shoulders of giants. Here are the third-party non-GPL libraries we make use of:

Mustn't admit that we are not listing something that we do have to - we have to be believe the list _is_ complete... :grinning:

Reply:
👍

Reply:
Done _?_

Reply:
Looks good, a bit lengthy, how about this:

> Mudlet is built up the shoulders of others in the FOSS world; as well as making use of many GPL components we also make use of third-party non-GPL software:

I don't think it is interesting to know what needs to be mentioned and what doesn't need to be mentioned.

h3 is also supposed to be a header and this is too big to be a header, so how about just bolding it?

Reply:
Well "making use" is almost repeated with "make use" in the same sentence - so I'll use:

>Mudlet is built upon the shoulders of other projects in the FOSS world; as well as using many GPL components we also make use of some third-party non-GPL software:

I'll check the formatting again, IIRC \<h4> is the next one down and that is left aligned rather than centred so that is probably not an alternative - mayhaps the shorter bit will look okay but if not I see what else works...

Reply:
Yeah perfect! Thanks!